test/50-assert-16-refute_output.bats

```diff
@@ -25,9 +25,9 @@ load test_helper
   [ "${lines[2]}" == '--' ]
 }
 
-@test 'refute_output(): reads <unexpected> from STDIN' {
-  run echo 'a'
-  run refute_output <<INPUT
+@test 'refute_output() - : reads <unexpected> from STDIN' {
+  run echo '-'
```

Comment:
As a stronger assertion that `-` as input on stdin doesn't cause problems. (Since the `-` is also the flag to the assertion)